docs/_posts/2020-01-08-533-release.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: blogpost
+title:  "5.3.3 Release Announcement"
+section_id: blog
+date:   2020-01-08 00:00:00
+author: Open GEE Development Team
+---
+
+<br />
+
+Hello Open GEE Community!
+ 
+We are happy to announce the official release of Open GEE 5.3.3!  This release updates Open GEE with minor enhancements and bug fixes for GEE Fusion and Server.
+
+**Enhancements:**
+
+**- Ability to set default database from admin page.** There is now an additional option for 3D databases that allows the user to specify a database as the "default". This default database will be served to the client if the server URL is specified without a database name.
+
+**- View information via getop while gesystemmanager is busy.**  Previously, the utility "getop" could not display information while the system manager was performing major calculations. This restriction has now been removed for most calculations. 
+
+**- Improved memory usage reporting.**  The memory usage estimations for asset and asset version caches have been improved and are now much more accurate. 
+
+**- Miscellaneous security updates.** Supplemental security updates have been made to protect against a variety of potential risks.
+
+**- Documentation improvements.**  Existing documentation has been refined and clarified. Documentation has been added for use of LDAP and for building on FIPS-enabled machines. 
+
+**- Portable and WMS clients.** Improved ease of use of Portable for WMS clients.
+
+**Upgraded components:**
+
+**- Use of Python 2.7 with EL6-based operating systems.**  EL6 OSs now use Python 2.7 in a virtual environment for increased security. EL7 and Ubuntu are unaffected.
+
+To download this release see the [release page](https://github.com/google/earthenterprise/releases/tag/5.3.3-XXXX.XX). The full release notes can also be found [here](http://www.opengee.org/geedocs/5.3.3/answer/releaseNotes/relNotesGEE5_3_3.html).
+ 
+Our thanks go out to all of the contributors who helped make this release possible! The next release, Open GEE 5.3.4, is already in progress!
+ 
+Would you like to be part of the project? Please join us on [Slack](http://opengee.slack.org/) and visit [the project’s Github repository](https://github.com/google/earthenterprise). We would love to see you there!
```

Comment:
That URL shows as for sale. @tst-lsavoie did you give this to someone on the listserv recently?

Reply:
There's been some chatter about how the slack link is down. We've pinged the maintainer of the website but haven't heard back.

Reply:
Basil's link goes to a good login page, so maybe it's back up. http://opengee.slack.com/ it is

Reply:
I did some digging. The "correct" URL is http://slack.opengee.org/ (not http://opengee.slack.org/). http://slack.opengee.org/ is not for sale, but it also never loads. http://opengee.slack.com/ works if you already have an account but doesn't help much if you don't. http://slack.opengee.org/ gave you a way to create an account when it worked.

I guess for this release http://opengee.slack.com/ is the best option since it at least has some utility until we get http://slack.opengee.org/ back up.